gson/src/main/java/com/google/gson/internal/LinkedHashTreeMap.java

```diff
@@ -762,6 +762,9 @@ void add(Node<K, V> node) {
     Node<K, V> lastReturned = null;
     int expectedModCount = modCount;
 
+    LinkedTreeMapIterator() {
```

Comment:
Right. A default constructor always exists, it just matches the class visibility by default. So in this case a second constructor had to be generated for the other class to create an instance. The change just promotes the implicit default constructor to a visibility that can be called directly.